assets/src/setup/components/options-context-provider.js

```diff
@@ -41,7 +40,7 @@ export function OptionsContextProvider( { children, optionsKey, optionsRestEndpo
 		setSavingOptions( true );
 
 		try {
-			await apiFetch( { method: 'post', url: optionsRestEndpoint, data: { [ optionsKey ]: options } } );
+			await apiFetch( { method: 'post', url: optionsRestEndpoint, data: options } );
```

Comment:
Shouldn't this be `GET`?

```suggestion
			await apiFetch( { method: 'get', url: optionsRestEndpoint, data: options } );
```

Reply:
This is `saveOptions` so needs to hit the `update_items` callback in the REST endpoint, which is registered with `WP_REST_Server::EDITABLE`. Let me know if I'm missing something, e.g. if you're suggesting a change to the REST endpoint.

Reply:
Ah, you're right. I thought this was retrieving the options.